File: Python/Mahasiswa.py
# import parent nya
from SivitasAkademik import SivitasAkademik

class Mahasiswa(SivitasAkademik) :
    # Variabel Protected
    _nim = ''
    _prodi = ''
    _fakultas = ''
    
    # Konstruktor
    def __init__(self, nik = "", nama = "", jenis_kelamin = "", asalUniv = "", emailEdu = "", nim = "", fakultas =""):
        # Mengakses variabel yang terdapat pada ortunya
        super().__init__(nik, nama, jenis_kelamin, asalUniv, emailEdu)
        # ini punya kelas Mahasiswa
        self._nim = nim
        self._fakultas = fakultas
    
    # Getter
    def get_nim(self) :
        return self._nim
    def get_fakultas(self) :
        return self._fakultas
    
    # Setter
    def set_nim(self, nim):
        self._nim = nim
    def set_nim(self, fakultas):
        self._fakultas = fakultas
        
        

File: C++/Mahasiswa.cpp
using namespace std;

//Mahasiswa adalah child, sivitasAkademik adalah parent
class Mahasiswa : public SivitasAkademik{
    // variabel yang dimiliki kelas ini
    protected:
        string nim;
        string fakultas;

    public:
        //konstruktor
        // Mahasiswa(){}
        // konstruktor sekaligus mengakses variabel parentnya
        Mahasiswa(string nik,string nama, string jenis_kelamin,string asal_universitas, string email_edu, string nim, string fakultas) : SivitasAkademik(nama, nik, jenis_kelamin, asal_universitas, email_edu){
            this->nim = nim;
            this->fakultas = fakultas;
        }

        // Getter dan setter
        // Getter
        string getNim() {
            return this->nim;
        }
        string getFakultas() {
            return this->fakultas;
        }

        // Setter
        void setNim(string nim) {
            this->nim = nim;
        }
        void setFakultas(string fakultas) {
            this->fakultas = fakultas;
        }

        // Destruktor
        ~Mahasiswa(){}
};